### DISCUSSION TITLE
About sampler setting

### DISCUSSION THREAD

**Discussion by ChiaYen-Kan** [vladmandic/sdnext#2479] [Q&A]
Hi

i have some question is about sampler setting

Q1.
how to set "brownian noise" is enabled in default?
i can't find the setting in "User Interface" 

Q2.
why the second pass don't have "Sampler options" and "Sigma algorithm" setting?
![sampler-setting](https://github.com/vladmandic/automatic/assets/40080620/89b0cdca-7cf7-4f81-9c96-eb257ea9af43)

Q3.
how to use "x/y/z grid" test  "Sampler options" and "Sigma algorithm" setting?

thank you

**Comment by Tillerz** [comment]
I think that is related to #2463 - some fields are really new and the loading/saving from default and/or previous session is not up to date with all the fields.

There seems to be a bug with reloading the previous session at all, as soon as this is working again I will make a list for Vlad with all the fields that are not working, and I guess the same fields are also missing for UI Settings / saving as default.

**Comment by vladmandic** [comment]
sampler options marked in red in the first image are actually stored as server settings. so they apply to primary and secondary sampler.
and they don't appear in ui settings since they're part of server settings.

**Reply by ChiaYen-Kan** [reply]
hi

based on @Tillerz and @vladmandic description:

1
some setting in current system is store in server side, that can't be change by ui-config.json, only change it by browser every time when restart server and browser

2
first pass and second is share "Sampler options" and "Sigma algorithm" setting, so i can't set difference setting in second pass

3
when some setting is stored in server side, x/y/z grid can't change it

Is my understanding correct?

thank you
